src/admin/pilot_env_on_cvmfs.rst

```diff
@@ -0,0 +1,42 @@
+Pilot Environment on CVMFS
+==========================
+
+To make sure pilot can run with all required environments, we maintained a pilot environment to
+deploy necessary packages to cvmfs.
```

Comment:
Suggestion:
To make sure that the pilot can run in various environments across sites, we maintain the pilot environment by
deploying necessary packages to cvmfs.